frontend/src/common/PageWithTabs.tsx

```diff
@@ -110,30 +112,49 @@ export default function PageWithTabs({
 
   return (
     <>
-      <Stack
-        direction={{ xs: 'column', sm: 'row' }}
-        divider={<Divider flexItem orientation='vertical' />}
-        alignItems='center'
-        spacing={{ sm: 2 }}
-        sx={{ p: 2 }}
-      >
-        <Stack direction='row'>
-          <Typography component='h1' color='primary' variant='h6'>
-            {title}
-          </Typography>
-          {showCopyButton && (
-            <CopyToClipboardButton
-              textToCopy={textToCopy ? textToCopy : title}
-              notificationText='Copied to clipboard'
-              ariaLabel='copy to clipboard'
-            />
+      <Stack>
+        {sourceModelId && (
+          <Card
+            style={{
+              position: 'fixed',
+              left: 70,
+              right: 0,
+              top: 95,
+              backgroundColor: 'black',
+              borderRadius: 0,
+              color: 'white',
+            }}
+          >
+            <Typography justifyContent='center' align='center'>
+              [READONLY] Mirrored from {sourceModelId}
```

Comment:
I like making it clear where the model comes from but I'm not a big fan of using a banner for this, it's in danger of being over-used. Could we include this information next to the model name at the top or something?